stable/external-dns/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: external-dns
-version: 2.17.0
+version: 2.17.1
```

Comment:
bump minor version, it should be `2.18.0`

Reply:
There was another minor  revision in the interim.  Bumped to 2.19.0